src/main/java/emissary/place/MultiFileUnixCommandPlace.java

```diff
@@ -285,7 +288,7 @@ protected void getFileList(File tmpDir, String inputFileName, List<File> outFile
                     logger.debug("Ignoring file '{}' because it is the input file.", fname);
                     continue;
                 }
-                if (file.length() == 0) {
+                if (file.length() == 0 && ignoreEmptyFile) {
```

Comment:
Skip unnecessary file length check if the feature flag is false.
```suggestion
                if (ignoreEmptyFile && file.length() == 0) {
```